### PR TITLE
man: explain what kind of data is actually included in "systemd-analy…

### PR DESCRIPTION
**Pull request by poettering**
…ze plot" output

Prompted by: https://lists.freedesktop.org/archives/systemd-devel/2023-January/048736.html

### CODE REVIEW

File: man/systemd-analyze.xml [hunk 1]
```diff
@@ -284,6 +284,17 @@ Timestamp units-load-finish: Thu 2019-03-14 23:28:07 CET
 $ eog bootup.svg&amp;
 </programlisting>
       </example>
+
+      <para>Note that this plot is based on the most recent per-unit timing data of loaded units. This means
+      if – after boot – a unit gets started, then stopped and then started again the information shown will
```
Comment (keszybz) [inline man/systemd-analyze.xml]: "This means *that* if"

Hmm, why "after boot"? If it gets stopped and started again during boot, we would still show only info about the latest invocation, iiuc.

File: man/systemd-analyze.xml [hunk 2]
```diff
@@ -284,6 +284,17 @@ Timestamp units-load-finish: Thu 2019-03-14 23:28:07 CET
...
+      cover the most recent start cycle, not the first one. Thus it's recommended to consult this information
+      only shortly after boot, so that this distinction doesn't matter. Moreover, units that are not
+      referenced by any other unit through a dependency might be unloaded by the service manager once they
+      terminate (and did not fail). Such units will not show up in the plot, hence. Thus it is essential to
```
Comment (keszybz) [inline man/systemd-analyze.xml]: Strike ", hence".

File: man/systemd-analyze.xml [hunk 3]
```diff
@@ -284,6 +284,17 @@ Timestamp units-load-finish: Thu 2019-03-14 23:28:07 CET
...
+      ensure that all relevant units are referenced directly or indirectly from a unit such as
+      <filename>multi-user.target</filename> which is generally what is activated at boot. Note that
+      typically this is the target unit that regular units are hooked into anyway (directly or indirectly),
+      hence this requirement typically does not require any extra work, but effectively is given.</para>
```
Comment (keszybz) [inline man/systemd-analyze.xml]: Dunno, this suggests that people should create dependencies solely for the purpose of systemd-analyze. Maybe it'd be better to just explain that units that are not pulled in through a dependency will not show up in the plot and leave it at that?

**Review by keszybz — CHANGES_REQUESTED** [review]

**Comment by poettering** [comment]
made the suggested changes. ptal.

**Comment by keszybz** [comment]
LGTM.

**Pull request merged**